### PR TITLE
Fix most warnings when generating documentation with Doxygen 1.8.20

### PR DESCRIPTION
**Pull request by sercero**
- Update version to 1.28 and copyright year to 2017 (as last version from sourceforge)
 - Fix most warnings from Doxygen (about missing parameters / wrong references)

### CODE REVIEW

File: oggsound/include/OgreOggISound.h [hunk 1]
```diff
@@ -1,14 +1,14 @@
 /**
 * @file OgreOggISound.h
 * @author  Ian Stangoe
-* @version 1.26
```
Comment (paroj) [inline oggsound/include/OgreOggISound.h:4]: ugh.. you opened a can of worms there..

All this boilerplate code on top is mostly nonsense:
- nobody cares about @file
- a single @author is wrong
- @version is better tracked in cmake
- for License it is better to reference the top-level file and only leave a SPDX identifier here. See the Ogre Coding Guidelines: https://github.com/OGRECave/ogre/blob/master/Docs/CodingStandards.md#top-level-organisation-issues and https://spdx.dev/ids/

Comment (paroj) [reply oggsound/include/OgreOggISound.h:4]: but I guess it is too much hassle to remove the non-doxygen warning related changes now, so we should go with this as is?

Comment (sercero) [reply oggsound/include/OgreOggISound.h:4]: I think it should go as is to respect the original author mostly.

But if you want, I can remove the @file and @version

About the license the problem is that I don't know if the text provided is MIT or other license, sound like MIT...

Comment (paroj) [reply oggsound/include/OgreOggISound.h:4]: > I think it should go as is to respect the original author mostly.

I keep the original author in the "Copyright (c)" line of the [LICENSE](https://github.com/OGRECave/ogre-audiovideo/blob/master/oggsound/LICENSE) file for this reason.
Oggsound is MIT as you assumed, but ogrevideo is BSD-3

> But if you want, I can remove the @file and @Version

yes please

**Pull request merged**